app/services/testdata/GeneratorConfig.scala

```diff
@@ -17,9 +17,10 @@
 package services.testdata
 
 import model.EvaluationResults.Result
+import org.joda.time.LocalDate
```

Comment:
Maybe I'm wrong of just dream of it, but I read somewhere to stop using Joda. Maybe Olly knows a bit more?